earhorn/main.py

```diff
@@ -98,6 +99,14 @@
     ),
     is_flag=True,
 )
+@click.option(
+    "--s3-archive",
+    envvar="S3_ARCHIVE",
+    help=(
+        "Specifying wheter to save to s3 bucket or not"
+    ),
+    is_flag=True,
+)
```

Comment:
Maybe we could use the archive path and check if it starts with s3:// to enable s3 storage.